spec/support/command_helpers.rb

```diff
@@ -271,18 +271,20 @@ def set_security_code level
     send_command_and_confirm @task, command_list, "intention to set security code"
   end
 
-  def set_date
+  def set_date date
     @site.log "Set date", level: :test
     command_list = build_command_list :M0104, :setDate, {
       securityCode: SECRETS['security_codes'][1],
-      year: 2020,
-      month: 9,
-      day: 29,
-      hour: 17,
-      minute: 29,
-      second: 51
+      year: date.year,
+      month: date.month,
+      day: date.day,
+      hour: date.hour,
+      minute: date.min,
+      second: date.sec
     }
-    send_command_and_confirm @task, command_list, "intention to set date"
+
+    message, result = send_command_and_confirm @task, command_list, "intention to set date"
+    return message, result
```

Comment:
the two lines can be reduced to just:
send_command_and_confirm @task, command_list, "intention to set date"